packages/dev/core/src/Engines/WebGPU/webgpuTintWASM.ts

```diff
@@ -52,15 +49,11 @@ export class WebGPUTintWASM {
         }
 
         if (twgslOptions.jsPath && twgslOptions.wasmPath) {
-            if (IsWindowObjectExist()) {
-                await Tools.LoadScriptAsync(twgslOptions.jsPath);
-            } else {
-                importScripts(twgslOptions.jsPath);
-            }
+            await Tools.LoadScriptAsync(twgslOptions.jsPath);

```

Comment:
You missed wrapping `jsPath` in a `Tools.GetScriptUrl()` call.

Reply:
The function does it internally